assembl/static2/js/app/components/svg/disagree.jsx

```diff
@@ -26,8 +26,8 @@ class Disagree extends React.Component {
             1.39072848 14,
             1.39072848 L14,
             1.39072848 Z"
-            stroke="#F75959"
-            fill="#FFFFFF"
+            stroke={color || '#F75959'}
+            fill={backgroundColor || '#ffffff'}
```

Comment:
you should probably use default props for all these (same for `dontUnderstand.jsx` and `like.jsx` and `moreInfo.jsx`)